api/src/main/java/com/codeforcommunity/dto/report/ReportGeneric.java

```diff
@@ -123,4 +123,39 @@ public String getVisitReason() {
   public void setVisitReason(String visitReason) {
     this.visitReason = visitReason;
   }
+
+  public String toHeaderCSV() {
+    return "Report ID,"
+        + "Created At"
+        + "Updated At"
+        + "School ID"
+        + "User ID"
+        + "Number of Children"
+        + "Number of Books"
+        + "Most Recent Shipment Year"
+        + "Library Status\n";
```

Comment:
Should each of these segments have a comma separating them? Could we use a for loop + StringBuilder?

Reply:
aaaa good catch definitely needs a comma